frappe/utils/global_search.py

```diff
@@ -373,11 +373,57 @@ def sync_global_search():
 	:param flags:
 	:return:
 	"""
-	while frappe.cache.llen("global_search_queue") > 0:
-		# rpop to follow FIFO
-		# Last one should override all previous contents of same document
-		value = json.loads(frappe.cache.rpop("global_search_queue").decode("utf-8"))
-		sync_value(value)
```

Comment:
deprecate old unused functions with `@deprecated`. We usually remove them after some time has passed or before next major release.

Reply:
Got it, although if you're referring to `sync_value` it's used elsewhere.